app/views/admin/settings/index.html.slim

```diff
@@ -1,27 +1,48 @@
-- content_for(:title, 'Settings')
+- content_for(:title, t("settings.settings"))
 
 script type="text/javascript" src="https://appcenter.intuit.com/Content/IA/intuit.ipp.anywhere-1.3.3.js"
 
 javascript:
   intuit.ipp.anywhere.setup({menuProxy: '/path/to/blue-dot', grantUrl: '#{authenticate_admin_accounting_quickbooks_url}'});
 
 section.block
+  h2 = t("quickbooks.connection.connection")
+
   .form-horizontal
     .form-group
-      label.col-sm-2 Quickbooks Status:
+      label.col-sm-4.text-right = t("quickbooks.connection.status")
       - if Division.root && Division.root.quickbooks_connected?
-        .col-sm-10 Connected
+        .col-sm-8 = t("quickbooks.connection.connected")
       - else
-        .col-sm-10 Not Connected
+        .col-sm-8 = t("quickbooks.connection.not_connected")
 
     - if Division.root && Division.root.quickbooks_connected?
       .form-group
-        label.col-sm-2 Disconnect from Quickbooks:
-        .col-sm-10 
-          a href="#{disconnect_admin_accounting_quickbooks_path}" Disconnect
+        label.col-sm-4.text-right = t("quickbooks.connection.disconnect_from_qb")
+        .col-sm-8
+          a href="#{disconnect_admin_accounting_quickbooks_path}" = t("quickbooks.connection.disconnect")
     - else
       .form-group
-        label.col-sm-2 Connect to Quickbooks:
-        .col-sm-10
+        label.col-sm-4.text-right = t("quickbooks.connection.connect_to_qb")
+        .col-sm-8
```

Comment:
Actually now that I look at this more there is something a bit troubling. We are replicating the HTML for a form which is not very DRY. Imagine in the future we want to change the form template a bit (quite likely), we will forget about this one-off duplication and this piece of the form will not match.

Here is an idea: add these two 'fields' to the form below using the simple form block syntax (not all form fields need to be actual fields...) Then we get the styling for free.